modules/configuration/pages/allow_and_deny_lists.adoc

```diff
@@ -0,0 +1,43 @@
+= Configure an Allow or Deny List
+:description: Learn how to configure an allow or deny list for a Redpanda Connect instance.
+
+You can configure an allow or deny list to limit the Redpanda Connect components that users can run within data pipelines on a Redpanda Connect instance.
+
+Introduced in version 4.39.0.
+
+== Limitations
+
+- You can configure only one allow list, or one deny list per Redpanda Connect instance.
+- When you add the name of a component to your allow or deny list, you grant or block access to all related components. For example, if you include `aws_s3` in your list, you grant or block access to the entire `aws_s3` family of components, including the cache, input, and output.
+
+== Configure an allow list
+
+Configure an allow list to let users run only a subset of named components on your Redpanda Connect instance.
```

Comment:
Both this intro sentence and the one on line 31 seem repetitious. You've already explained the function each list type serves in the intro, so you could just have:
To configure an allow list: 
or
To configure a deny list:

Your call.

Reply:
Updated as requested.